Dockerfile

```diff
@@ -42,9 +42,9 @@ RUN apk add --no-cache bash openssh ca-certificates jq curl openssl perl git zip
  && apk add --no-cache bash-completion \
  && update-ca-certificates
 
-ARG JP_VERSION="0.1.3"
+ARG JP_VERSION="0.2.1"
```

Comment:
There is no breaking change from 0.1.3 to 0.2.1
0.2.0 add `jp-linux-arm64` release.